demo/ServiceHook.fs

```diff
@@ -0,0 +1,62 @@
+module Demo.ServiceHook
+
+open Feliz
+open Browser
+open Fable.Core.JsInterop
+open Glutinum.Feliz.ReactGoogleAutocomplete
+open type Glutinum.Feliz.ReactGoogleAutocomplete.Exports
+open Fable.Core.JS
+
+[<ReactComponent>]
+let private Component () =
+    let response =
+        usePlacesAutocompleteService (
+            UsePlacesAutocompleteServiceConfig(
+                apiKey = VITE_GOOGLE_MAP_API_KEY,
+                debounce = 300
+            )
+        )
+
+    let handleOnChange =
+        fun (e: Types.Event) ->
+            let target = e.target
+            let input = target?value
+
+            response.getPlacePredictions (
+                {|
+                    input = input
+                |}
+            )
```

Comment:
If I remember correctly, Feliz has a version of `prop.onChange` which gives you access to the target value directly.